src/Pages/Home/Home.js

```diff
@@ -1,16 +1,112 @@
-import React from "react";
+import React, { useEffect, useState } from "react";
 import Hero from "../../Components/Hero/Hero";
 import ListenOn from "../../Components/ListenOn/ListenOn";
 import LatestEpisodes from "../../Components/LatestEpisodes/LatestEpisodes";
 import HostsSection from "../../Components/HostsSection/HostsSection";
 import Patreon from "../../Components/Patreon/Patreon";
+import axios from "axios";
+
+var XMLParser = require("react-xml-parser");
+
+function convertUnicode(myString) {
+    let convertedString = myString.replaceAll("%20", " ");
+    return convertedString;
+}
+
+function stringChoppy(mystring) {
+    let slicedString = mystring.slice(0, mystring.length - 2);
+
+    return slicedString;
+}
+
+function convertDate(date) {
+    let dateArray = date.split(" ");
+    let end = "";
+    let day = "";
+    let digitOne = Number(dateArray[1][0]);
+    let digitTwo = Number(dateArray[1][1]);
+
+    if (digitOne === 0) {
+        day = digitTwo;
+    } else day = dateArray[1];
+
+    if (digitTwo === 1) {
+        end = "st";
+    }
+    if (digitTwo === 2) {
+        end = "nd";
+    }
+    if (digitTwo === 3) {
+        end = "rd";
+    }
+    if (digitTwo >= 4) {
```

Comment:
You may want to make all of these `if` into `else if` from line 36 down. Or you could use a switch case.
Because you only ever want it to check for one of them and not all of them. When they are all `if` it will check each one even if it already found an answer on the first `if`. But if they are `else if` it will ignore them as soon as it finds the right choice. A switch case works a similar way to `if` `else if`